lms/djangoapps/ccx/views.py

```diff
@@ -572,6 +576,28 @@ def ccx_course(ccx_locator):
     yield course
 
 
+def add_master_course_staff_to_ccx(course, ccx_key, display_name):
+    """
+    Assign staff of master course, staff role on ccx.
+    :param course: Master course instance
+    :param ccx_key: CCX course key
+    :param display_name: ccx display name for email
+    """
+    list_staff = list_with_level(course, 'staff')
+    with ccx_course(ccx_key) as course:
+        email_params = get_email_params(course, auto_enroll=True, course_key=ccx_key, display_name=display_name)
+        for staff in list_staff:
+            allow_access(course, staff, 'staff')
+            # Enroll the coach in the course
```

Comment:
you are enrolling not only the coach, but any other staff member, if I understood well